### ISSUE TITLE
Sharing a file to a room results in increased unread marker

### ISSUE THREAD

**Issue by SystemKeeper** [nextcloud/spreed#9488]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Upload a picture/file

### Expected behaviour
Since I am sharing that file, the unread marker should not increase for myself.

### Actual behaviour
The unread marker increases. This happens as soon as I share a file to a conversation:
```
curl 'https://sermo.nextcloud.com/ocs/v2.php/apps/files_sharing/api/v1/shares' \
  -H 'authority: sermo.nextcloud.com' \
  -H 'accept: application/json, text/plain, */*' \
  -H 'content-type: application/json' \
  -H 'cookie: <cookie>' \
  -H 'origin: https://sermo.nextcloud.com' \
  -H 'requesttoken: <token>' \
  --data-raw '{"shareType":10,"path":"//Talk/Bildschirmaufnahme 2023-05-06 um 14.25.51.mov","shareWith":"1234","referenceId":"asdf","talkMetaData":"{\"messageType\":\"\"}"}' \
  --compressed
```
(just copied from the browser). Each time this is executed, the file gets shared to the conversation and increases the unread marker. This can also be reproduced from talk-ios when sharing a file from outside of the app (go to photo library and then share with Nextcloud Talk, after opening the app you have unread messages).
Not 100% sure, but this wasn't the case in older versions, was it?

Note: This also happens when creating a Poll for example and directly leaving the conversation after creating it. When the sidebar refreshes, I have now 1 unread message in that conversation, which is my poll.

## Talk app
sermo, master. Also seen on latest 26.

**Comment by nickvergessen** [comment]
yeah well, the problem is we can't set the read marker all the time, because maybe you are sharing from the files app, etc. and didn't actually read all messages in the conversation before.

But I can see how it is unexpected from talk UI when navigating away afterwards quickly.
